qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -65,31 +60,22 @@
         "PMD.TooManyMethods", "PMD.AvoidDuplicateLiterals", "PMD.GodClass"
     }
 )
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 final class CheckstyleValidatorTest {
 
     /**
-     * Name of property to set to change location of the license.
+     * Local Common (JavaHelper pattern) with useful utils to test checkstyle.
      */
-    private static final String LICENSE_PROP = "license";
+    private Common common;
 
-    /**
-     * Directory with classes.
-     */
-    private static final String DIRECTORY = "src/main/java/foo";
-
-    /**
-     * License text.
-     */
-    private static final String LICENSE = "Hello.";
-
-    /**
-     * Rule for testing.
-     */
-    private License rule;
+    @BeforeAll
```

Comment:
The simple way to make the same without before all - just create a base class with all logic (including BeforeEach).
For now it is not very clear, why should we always work via common and what is that common

Reply:
It seemed like a great idea to me! Added a common CheckstyleTestBase class for common constants and utilities